podpac/core/node.py

```diff
@@ -1258,7 +1263,7 @@ def _process_kwargs(name, d, definition, nodes):
         kwargs[k] = v
 
     for k, v in d.get("inputs", {}).items():
-        kwargs[k], nodes = _lookup_input(nodes, name, v, definition)
+        kwargs[k]= _lookup_input(nodes, name, v, definition)
```

Comment:
does `_lookup_input` change nodes? I do prefer returning both if that's the case. Otherwise a comment in the code would be quite helpful.